resources/js/components/inputs/relationship/RelationshipInput.vue

```diff
@@ -69,8 +69,6 @@
                     :selections-url="selectionsUrl"
                     :site="site"
                     :initial-columns="columns"
-                    initial-sort-column="title"
-                    initial-sort-direction="asc"
```

Comment:
Removing these lines caused the arrows in the table headers to disappear so you couldn't see which column was being sorted. (Even though it was being sorted correctly.)